src/components/database/Queries.js

```diff
@@ -0,0 +1,345 @@
+// @flow
+import {database, fieldValue} from '../../firebaseFeatures.js';
+import type {UserType, PendingType, ActionType} from '../admin/FlowTypes.js';
```

Comment:
Is there a reason that FlowTypes file belongs to admin directory? I see it's used database and admin directories, so maybe it would be better to move it to /components (or create utils/types directory)?

Reply:
yes this is a good idea